app/controllers/admin/api/base_controller.rb

```diff
@@ -131,6 +131,7 @@ def authorize_service_plans!
 
   ## BackendAPIs
 
+  ##~ @parameter_backend_api_id_by_id = { :name => "id", :description => "ID of the backend API.", :dataType => "int", :required => true, :paramType => "path" }
```

Comment:
```suggestion
  ##~ @parameter_backend_api_id_by_id = { :name => "id", :description => "ID of the backend.", :dataType => "int", :required => true, :paramType => "path" }
```

See https://github.com/3scale/porta/pull/1254